xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -239,9 +239,6 @@ bool CDVDDemuxFFmpeg::Open(const std::shared_ptr<CDVDInputStream>& pInput, bool
   m_pFormatContext  = avformat_alloc_context();
   m_pFormatContext->interrupt_callback = int_cb;
 
-  // try to abort after 30 seconds
-  m_timeout.Set(30000);
```

Comment:
I would change the timeout only for realtime streams. Donˋt forget the users who want to play rar files from a raspberry pi over some slow network connection.

Reply:
Ah yeah nice idea, will do.

Reply:
I changed the timeout of first codepath to 15s per call, which equals the 30sec as we do 2 calls.
For mms files it's 15s because only 1 call is done if I read it correctly.
I hope that's rather OK at least I'd already press stop if start of playback takes more than 10sec.
If there are objections I can raise it to whatever value we are comfortable with.

Reply:
You change the timeout from 30 to 15 in beta stage. In this case you should give a reason why you think elupus was wrong setting it to 30 https://github.com/xbmc/xbmc/commit/9cd4a305f24830fc14582e08307bb58da1f4200b

Reply:
Fixed, I'll keep the 30s.